src/core/components/operation.jsx

```diff
@@ -151,7 +151,7 @@ export default class Operation extends PureComponent {
                     authActions.showDefinitions(applicableDefinitions)
                   }}
                 />
-            }
+            } */}
```

Comment:
Do we still want this commented out here? If so, can you please add a note as to why it was commented out, and why you'd want to uncomment it?

Reply:
this displays the lock icon, indicating if the action requires authorization. it's showing as 'unlocked' in our APIs which isn't correct. this could be re-enabled if fixed in the generated swagger docs (although all of our endpoints require auth, so it's not very useful IMO)